Syntaxes/AppleScript.tmLanguage

```diff
@@ -8,7 +8,7 @@
 		<string>script editor</string>
 	</array>
 	<key>firstLineMatch</key>
-	<string>^#!.*(osascript)</string>
+	<string>^#!.*(osascript)(?!(?i:javascript)</string>
```

Comment:
This would only match `javascript` when it is right beside osascript, it would appear later in the line after the `-l` option correct? Also the parenthesis are unbalanced here.